elide-datastore/elide-datastore-aggregation/src/test/java/com/yahoo/elide/datastores/aggregation/AggregationDataStoreTransactionTest.java

```diff
@@ -74,8 +76,10 @@ public static void beforeAllTests() {
     }
 
     @BeforeEach
-    public void setUp() {
-        when(queryEngine.beginTransaction()).thenReturn(qeTransaction);
+    public void setUp(TestInfo info) {
+        if (!info.getTags().contains("SkipBeforeEach")) {
```

Comment:
Can you explain what this is for?

Reply:
one of test case was failing for "Extra Stubbing" error after the changes, so had to skip "BeforeEach" method for it to pass.

Reply:
removed this logic and added annotation "@MockitoSettings(strictness = Strictness.LENIENT)"